scripts/alltools.tcl

```diff
@@ -437,3 +445,17 @@ proc ordnumber {string} {
   }
   return $string
 }
+
+proc ucwords {string} {
+	list asent
+	foreach word [split [join $string]] {
+		lappend asent [string totitle $word]
```

Comment:
`string totitle` already lowercases the non-first characters, so ucwords and uclwords are identical right now. if you want to change the case of the first letter but leave uppercase in the rest, you cannot use `[string totitle]` for that